test/e2e/otlp-sink/helm-chart/templates/collector-config-map.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
```

Comment:
This is basically unchanged, I only split test-resources/otlp-sink/otlp-sink.yaml into separate Helm templates (config map, service deployment etc.)